src/webgpu/util/math.ts

```diff
@@ -790,6 +840,11 @@ export function quantizeToI32(num: number): number {
   return i32(num).value as number;
 }
 
+/** @returns the closest 32-bit signed integer value to the input */
```

Comment:
In src/webgpu/shader/execution/expression/expression.ts there are probably other locations that are doing Math.trunc/Math.round for quantizing to u32 that should be converted to use this

Reply:
I'll follow up with that.